### PR TITLE
Release v0.1.4

### PR DESCRIPTION
**Pull request by Jack-Fraser16**

### CODE REVIEW

**Comment by radamson** [comment]
@Jack-Fraser16 Can you add `(Work in Progress)` to the SMART STU2 Test Suite title before this gets merged? See FI-1618

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# 0.1.4
+
+Note: This kit contains separate suites for both the STU1 and STU2 versions of the SMART App Launch Framework. However, development is ongoing and the SMART STU2 suite is not yet fully complete.
```
Comment (Jammjammjamm) [inline CHANGELOG.md:3]: Please wrap long lines.

File: lib/smart_app_launch/smart_stu2_suite.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 module SMARTAppLaunch
   class SMARTSTU2Suite < Inferno::TestSuite
     id 'smart_stu2'
-    title 'SMART App Launch STU2'
+    title '(Work in Progress) SMART App Launch STU2'
```
Comment (Jammjammjamm) [inline lib/smart_app_launch/smart_stu2_suite.rb:13]:
```suggestion
    title 'SMART App Launch STU2 (Work in Progress)'
```

File: lib/smart_app_launch/version.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (Jammjammjamm) [inline lib/smart_app_launch/version.rb:1]: We don't need to add this.

**Review by arscan — APPROVED** [review]

**Pull request merged**